src/System.Runtime.WindowsRuntime/src/System.Runtime.WindowsRuntime.csproj

```diff
@@ -9,7 +9,7 @@
     <NoWarn>$(NoWarn);1698;0436</NoWarn>
     <ProjectGuid>{844A2A0B-4169-49C3-B367-AFDC4894E487}</ProjectGuid>
     <Configurations>netcoreapp-Windows_NT-Debug;netcoreapp-Windows_NT-Release;netstandard-Debug;netstandard-Release;netstandard1.0-Debug;netstandard1.0-Release;netstandard1.2-Debug;netstandard1.2-Release;uap-Windows_NT-Debug;uap-Windows_NT-Release;uapaot-Windows_NT-Debug;uapaot-Windows_NT-Release</Configurations>
-    <DefineConstants>$(DefineConstants);SYSTEM_RUNTIME_WINDOWSRUNTIME_ASSEMBLY</DefineConstants>
+    <DefineConstants Condition="'$(TargetGroup)' == 'uapaot'">$(DefineConstants);SYSTEM_RUNTIME_WINDOWSRUNTIME_ASSEMBLY</DefineConstants>
```

Comment:
Because for netcoreapp we can’t use Environment from System.Runtime.Extensions since it conflicts because it is now a typeforward, and since this project has a reference from runtime to System.Private.Corelib, it conflicts with the new Environment type exposed there, so we can’t use the alias for it. 

However since the type for the version we have in corefx of ProjectN is internal in corelib, we need to resolve this type from System.Runtime.Extensions until we get a new ProjectN with the PR you just merged. We don’t use this constant anywhere else, right?

Reply:
I see, the compiler didn't like resolving the typeforward through the alias?  Seems like a that might be a compiler bug.

Reply:
> We don’t use this constant anywhere else, right?

Right.  Ok.